addons/mail/static/src/xml/discuss.xml

```diff
@@ -316,7 +316,8 @@
                     <span class="o_preview_counter">
                         <t t-if="preview.unreadCounter &gt; 0">&amp;nbsp;(<t t-esc="preview.unreadCounter"/>)</t>
                     </span>
-                    <span class="o_last_message_date"> <t t-esc="preview.date ? preview.date.fromNow() : ''"/> </span>
+                    <span class="o_discuss_icon o_thread_window_expand fa fa-expand" t-if="preview.isLinkedToDocumentThread"/>
```

Comment:
`t-if` before `class` (much easier to see that is `span` is not always in the DOM)